### PR TITLE
Style Transfer

### PR DESCRIPTION
**Pull request by iodapro**
This project is aiming to provide multi threading program with GUI interface for performing neural style transfer of the given content image using style images.

It is based on VGG19 reduced neural net (only convolutional layers are considered). In order to accomplish "style transfer" two loss functions are built:
1) Loss function of features of content image and  modified image (initialy original and modified images are equal)  is considered to be mean square error.
2) Loss function of features of style image and modified image is considered to be mean square error of gram matrices of those features. 

On each iteration both loss functions are computed as functions of input to the VGG19, which means that instead of weights, features maps of input images are updated, causing changing of initial image.

The core is folded with simple gui, where one can select content and styles images (one can select several style images in order to create mixture of styles). It is also possible to stop image modifying with Stop button. One can also save the style and reuse it later. 

In order to allow user interact with GUI during program execution, additional thread is created when Start NST button is pressed. This thread is joined afterwards after finishing the program or pushing Stop/Exit button.

It is thought that program is cross-platform. The only bottleneck is separation symbols in the paths, which are different for UNIX and Windows. This is overcome using function identifying the platform and returning delimiter based on OS.

Created under Ubuntu 18.04 with Python 2.7.15.

To run the program:
python runMe.py

### CODE REVIEW

**Comment by slavakurilyak** [comment]
@iodapro Thanks for your contribution! Please integrate [MLflow](https://github.com/mlflow/mlflow) into your code base and rename this task to be Style Transfer part of computer vision.

**Comment by slavakurilyak** [comment]
@iodapro please move this task to [tasks/computer-vision/style-transfer](https://github.com/produvia/ai-platform/tree/master/tasks/computer-vision/style-transfer)
